packages/react-component-ref/test/RefForward-test.tsx

```diff
@@ -1,7 +1,7 @@
+import { RefForward } from '@stardust-ui/react-component-ref'
```

Comment:
I don't like that we have inconsistency between the imports in these tests vs the imports in the components' tests. Can we change this import or is there a reason why it is like this?

Reply:
We want to test public APIs and get rid of `src` alias: it creates issues with IDE and works only in Jest/Webpack.

/cc @kuzhelov